GVRf/Framework/framework/src/main/java/org/gearvrf/GVRViewManager.java

```diff
@@ -124,7 +124,7 @@ public void setMainScene(GVRScene scene) {
         }
     }
 
-    private void setMainSceneImpl(GVRScene scene) {
+    protected void setMainSceneImpl(GVRScene scene) {
```

Comment:
Please make it private again and call GVRViewManager.onSurfaceCreated instead. There is a good reason this method ends with **Impl**.

Reply:
Reason being it is sensitive and subject to the occasional tweaks. It should be all confined to the base class.